utilities/replace_versions.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2018 Google LLC
```

Comment:
Please add documentation about how to use this script with examples of typical usage.